.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,41 @@
+    attributes:
+      label: Actual Behavior
+  - type: checkboxes
+    id: env
+    attributes:
+      label: Environment
+      options:
+        - label: Linux (glibc)
+        - label: Linux (musl)
+        - label: macOS (local)
+        - label: Other
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)